scripts/workspace_addjson.py

```diff
@@ -0,0 +1,87 @@
+"""
+Copyright 2018 IBM Corporation
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+
+import json, sys, os.path, argparse, codecs
+from cfgCommons import Cfg
+from wawCommons import printf, eprintf, getRequiredParameter
+
+
+try:
+    basestring            # Python 2
+except NameError:
+    basestring = (str, )  # Python 3
+
+nothingFound = True
+
+    # function to find a desired variable in complex json and add other part of json
+def includeJson(nodeJSON, keyJSON, keySearch, includeJSON):
+    global nothingFound
+    if keyJSON == keySearch:
+        nodeJSON[keyJSON] = includeJSON
+        nothingFound = False
+    # None
+    if nodeJSON[keyJSON] is None:
```

Comment:
this will probably raise `KeyError` exception when workspace json without `dialog_nodes` is provided, we should probably check that key exists (add `if` statement)

Reply:
and also add one test for that :|

Reply:
fixed. one test added